### DISCUSSION TITLE
Flags & Playlists

### DISCUSSION THREAD

**Discussion by Hamybal** [TT-ReBORN/Georgia-ReBORN#35] [Q&A] [answered]
Hey thank you so much for sharing this beauty!

Most of my music doesn't show the flag with artists, but just a few do.
What should I change to make it work on all songs?

I tried default options and customed options I can see under 'Playlist' self-made playlists. but I cannot see inside of it nor can I make play the songs inside of it?

Thanks in advance,
Have a wonderful day

**Comment by TT-ReBORN** [comment, marked as answer]
Hi Hamybal,

Some of your music files are tagged with %artistcountry%. If you want all of music to have the flag displayed you need to tag it with the %artistcountry% , i.e right click in the playlist on a cover to select the whole album, Properties, + add new. In the input field name write: ARTISTCOUNTRY and in the value field e.g United States

I don't know what you mean about the playlist, you need to explain it a bit more...

-TT
